src/language-js/print/statement.js

```diff
@@ -29,6 +34,16 @@ function printStatementSequence(path, options, print, property) {
     node[property].filter(({ type }) => type !== "EmptyStatement")
   );
 
+  const { originalText } = options;
+
+  if (
+    isClassBody &&
+    node.body.length > 0 &&
```

Comment:
Removing this condition does not trigger any test failures for me. I would like to investigate if it is really necessary. If so, then I would like to find a test for it.